5001-6000/LIT5354Mashaf.xml

```diff
@@ -62,19 +68,67 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          </langUsage>
       </profileDesc>
       <revisionDesc>
-         <change who="DE" when="2019-03-15">Created entity</change>         
+         <change who="DE" when="2019-03-15">created entity</change>
+         <change who="DN" when="2025-05-19">added incipit, bibliography, witnesses, work structure</change> 
+        
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <div type="bibliography"/>
-         <!---->
-         <listRelation>
-            <relation name="dcterms:creator"
-                      active="LIT5354Mashaf"
-                      passive="PRS12520TaklaH"/>
+         <div type="bibliography">
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Platonov2002Lies"/>                  
+               </bibl>
+            </listBibl>            
+            <listRelation>
+    
+            <relation name="dcterms:creator" active="LIT5354Mashaf" passive="PRS12520TaklaH"/>
+               <relation name="lawd:hasAttestation" active="LIT5354Mashaf" passive="IVEf27"/>
```

Comment:
you do not need hasAttestation for works, this is covered by listWit, hasAttestation is for other entities

Reply:
ok